### PR TITLE
Modelsandtextures

### PR DESCRIPTION
**Pull request by Jeronimo97**

### CODE REVIEW

File: src/main/java/net/gir/girsignals/EnumsHL.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package net.gir.girsignals;
+
+import net.minecraft.util.IStringSerializable;
+
+public class EnumsHL {
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/EnumsHL.java:5]: Merge with EnumsHV

File: src/main/java/net/gir/girsignals/EnumsKS.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package net.gir.girsignals;
+
+import net.minecraft.util.IStringSerializable;
+
+public class EnumsKS {
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/EnumsKS.java:5]: Merge with EnumsHV

File: src/main/java/net/gir/girsignals/blocks/SignalHL.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import net.minecraft.block.properties.PropertyBool;
+import net.minecraft.block.properties.PropertyEnum;
+import net.minecraftforge.common.property.Properties.PropertyAdapter;
+
+public class SignalHL extends SignalBlock {
+	
+	public SignalHL() {
+		super("HL");
+	}
+	public static final PropertyAdapter<HL> LICHTHAUPTSIGNAL = new PropertyAdapter<HL>(
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/blocks/SignalHL.java:14]: English names
```suggestion
	public static final PropertyAdapter<HL> STOPSIGNAL = new PropertyAdapter<HL>(
```

File: src/main/java/net/gir/girsignals/blocks/SignalHL.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+			PropertyEnum.create("lichthauptsignale", HL.class));
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/blocks/SignalHL.java:15]: English names
```suggestion
			PropertyEnum.create("stopsignal", HL.class));
```

File: src/main/java/net/gir/girsignals/blocks/SignalKS.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import net.minecraft.block.properties.PropertyBool;
+import net.minecraft.block.properties.PropertyEnum;
+import net.minecraftforge.common.property.Properties.PropertyAdapter;
+
+public class SignalKS extends SignalBlock {
+
+	public SignalKS() {
+		super("KS");
+	}
+	public static final PropertyAdapter<KS> KOMBISIGNAL = new PropertyAdapter<KS>(
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/blocks/SignalKS.java:15]: English names
```suggestion
	public static final PropertyAdapter<KS> STOPSIGNAL = new PropertyAdapter<KS>(
```

File: src/main/java/net/gir/girsignals/blocks/SignalKS.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+			PropertyEnum.create("kombisignale", KS.class));
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/blocks/SignalKS.java:16]:
```suggestion
			PropertyEnum.create("stopsignal", KS.class));
```

File: src/main/java/net/gir/girsignals/EnumsHL.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	public enum HL implements IStringSerializable {
+		None, Off;
+
+		@Override
+		public String getName() {
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/EnumsHL.java:10]: See new API

File: src/main/java/net/gir/girsignals/EnumsKS.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+	
+	public enum KS implements IStringSerializable {
+		None, Off; 
+
+		@Override
+		public String getName() {
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/EnumsKS.java:11]: See new API

**Review by MrTroble — CHANGES_REQUESTED** [review]

**Review by Jeronimo97 — COMMENTED** [review]
Jep

File: src/main/java/net/gir/girsignals/Enums.java [hunk 9]
```diff
@@ -2,21 +2,21 @@
 
 import net.minecraft.util.IStringSerializable;
 
-public class EnumsHV {
-	
+public class Enums {
```
Comment (MrTroble) [inline src/main/java/net/gir/girsignals/Enums.java:5]: Refactor to EnumSignals to remove ambiguity between java.util.Enum

**Review by MrTroble — CHANGES_REQUESTED** [review]

File: src/main/java/net/gir/girsignals/Enums.java (hunk 9 above)
Comment (Jeronimo97) [reply src/main/java/net/gir/girsignals/Enums.java]: Done

**Review by MrTroble — APPROVED** [review]

**Pull request merged**